cantools/database/utils.py

```diff
@@ -205,3 +205,14 @@ def create_little():
     return Formats(big_compiled,
                    little_compiled,
                    big_padding_mask & little_padding_mask)
+
+
+class Colors:
```

Comment:
I think the database utilities is a strange place for a class defining ANSI color escape sequences. Better move thisto the dump subparser?!

Reply:
You're right. I'll find a better place for it.

Reply:
I moved it to the dump.formatting module